### PR TITLE
Add flow type declarations

### LINKED ISSUE #32: Flow type support?
We actively use the library and really like it 👍 We've been trying to improve our flow coverage and were wondering if you could add flow types to the library?

Thanks!

### PR DESCRIPTION
**Pull request by sanjaypojo**
Closes #32

### CODE REVIEW

**Comment by sanjaypojo** [comment]
@marudy here's the PR that I had mentioned! Let me know if you want any changes :)

File: index.js.flow [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+@flow
```
Comment (marudy) [inline index.js.flow]: Can we do this one liner ? :)

**Comment by marudy** [comment]
@sanjaypojo thanks for the PR. Can you maybe add a quick note in README as well ? Also did you test your additions locally to see if they work as expected? If yes we can release it soon.

Thanks!

File: index.js.flow (hunk 1 above)
Comment (sanjaypojo) [reply index.js.flow]: Done!

**Comment by sanjaypojo** [comment]
@marudy I've tested the additions locally and it works fine! With regards to usage, flow types should just automatically work. Where in the readme should we add it? Didn't find anything for TS, but they should probably go together

**Comment by marudy** [comment]
@sanjaypojo merged! I'd put it some description in README under `The Package` -> `Updates`. If we add that as well, it will be awesome 👍

**Comment by marudy** [comment]
@sanjaypojo released new version 1.4.0 that contains this . Thanks for the help 👍

**Comment by sanjaypojo** [comment]
Awesome, thanks for the update @marudy :)

**Pull request merged**